azext_iot/common/_azure.py

```diff
@@ -241,15 +241,21 @@ def _find_iot_dps_from_list(all_dps, dps_name):
 
 
 def get_iot_central_tokens(cmd, app_id, central_api_uri):
-    def get_event_hub_token(app_id, iotcAccessToken):
+    def formatUrl(url):
+        finalUrl = url
+        if not finalUrl.startswith('https://'):
+            finalUrl = 'https://{}'.format(finalUrl)
+        return finalUrl
+
+    def get_tokens(app_id, iotcAccessToken, central_api_uri):
         import requests
-        url = "https://{}/v1-beta/applications/{}/diagnostics/sasTokens".format(central_api_uri, app_id)
+        url = "{}.{}/system/iothubs/generateSasTokens".format(formatUrl(app_id), central_api_uri)
```

Comment:
Is there any documentation around `system/iothubs/generateSasTokens` route? Is it a good idea to expose it here if its not a public/long term support API?

Reply:
This is the IoT Central gateway API.  It is long term, but not public.  The CLI is one of the reasons this endpoint was added so it is safe to use, the team behind it are aware we are using it here.